test/unit/range/views/view_kmer_hash_test.cpp

```diff
@@ -22,158 +22,105 @@
 using seqan3::operator""_dna4;
 using seqan3::operator""_dna5;
 using seqan3::operator""_shape;
+using result_t = std::vector<size_t>;
 
-class kmer_hash_test : public ::testing::Test
-{
-protected:
-    using result_t = std::vector<size_t>;
-
-    static constexpr auto ungapped_view = seqan3::views::kmer_hash(seqan3::ungapped{3});
-    static constexpr auto gapped_view = seqan3::views::kmer_hash(0b101_shape);
-
-    std::vector<seqan3::dna4> text1{"AAAAA"_dna4};
-    std::vector<seqan3::dna4> const ctext1{"AAAAA"_dna4};
-    result_t ungapped1{0,0,0};
-    result_t gapped1{0,0,0};
-
-    std::vector<seqan3::dna4> text2{"ACGTAGC"_dna4};
-    std::vector<seqan3::dna4> const ctext2{"ACGTAGC"_dna4};
-    result_t ungapped2{6,27,44,50,9};
-    result_t gapped2{2, 7, 8, 14, 1};
-
-    std::vector<seqan3::dna4> text3{"AC"_dna4};
-    std::vector<seqan3::dna4> const ctext3{"AC"_dna4};
-    result_t ungapped3{};
-    result_t gapped3{ungapped3};
-
-    seqan3::bitcompressed_vector<seqan3::dna4> text4{"ACGTAGC"_dna4};
-    seqan3::bitcompressed_vector<seqan3::dna4> const ctext4{"ACGTAGC"_dna4};
-    result_t ungapped4{ungapped2};
-    result_t gapped4{gapped2};
+static constexpr auto ungapped_view = seqan3::views::kmer_hash(seqan3::ungapped{3});
+static constexpr auto gapped_view = seqan3::views::kmer_hash(0b101_shape);
+static constexpr auto prefix_until_first_thymine = seqan3::views::take_until([] (seqan3::dna4 x)
```

Comment:
maybe put that into the type test class `kmer_hash_properties_test`

Reply:
You wrote me that you couldn't make this work. Since these are view adaptors, it is fine for me to put them here.

I'm wondering why `TextFixture::ungapped_view` wasn't working.